packages/mwp-app-route-plugin/src/index.js

```diff
@@ -20,14 +20,13 @@ export function register(
 	const ldClient = LaunchDarkly.init(options.ldkey || LAUNCH_DARKLY_SDK_KEY, {
 		offline: process.env.NODE_ENV === 'test',
 	});
-	server.expose('getFlags', memberObj => {
-		const key = (memberObj && memberObj.id) || 0;
-		return ldClient.allFlags({ ...memberObj, key, anonymous: key === 0 }).then(
+	server.expose('getFlags', (user: LaunchDarklyUser) => {
+		return ldClient.allFlags(user).then(
 			flags => flags,
 			err => {
 				server.app.logger.error({
 					err,
-					member: memberObj,
```

Comment:
If I remember right, `member` is a magic keyword for this log that will allow stackdriver to annotate the error with the particular member ID that generated the error - you can just make this `member: user`

Reply:
Done